### ISSUE TITLE
ReferenceError: require is not defined

### ISSUE THREAD

**Issue by linediconsine** [component-driven/cypress-axe#170]
Hi and thanks for develop and maintaining cypress-axe, I see an error with this versions

```
 "cypress-axe": "^1.5.0",
  "cypress": "^13.6.1",
  "axe-core": "^4.8.2",
```
  
  Accessibility test are failing with 
  
```
 [positive] Loader component should have no accessibility violations:

ReferenceError: require is not defined
at exports.injectAxe (http://localhost:6060/__cypress/tests?p=test/integration/support/index.ts:27:190)
```

I created this patch
https://github.com/component-driven/cypress-axe/pull/169

That seem to solve the problem

**Comment by aneta-kjeldsen** [comment]
I'm experiencing the same error. Is it possible to merge the PR with the fix.
